test/test_autograd.py

```diff
@@ -5006,7 +5006,7 @@ def run_functional_checks(test_case, test_name, name, apply_fn, run_grad_checks,
                 'eq_', 'ne_', 'add', '__radd__', 'sum', 'conj', 'sin', 'cos', 'mul', 'sinh',
                 'cosh', '__rmul__', 'sgn', 'abs', 'dot', 'vdot', 'tensor_split', 'matmul',
                 'bmm', 'mv', 'ger', 'diagonal', 'atan', 'angle', 'tanh', 'fill_', 'sub',
-                'exp', 'mean', 'addcmul', 'addcdiv'] + separate_complex_tests
+                'exp', 'mean', 'inverse', 'addcmul', 'addcdiv'] + separate_complex_tests
```

Comment:
Maybe this should just be a list to avoid merge conflicts lol:
```
complex_list = [
    't',
    'view',
    'reshape',
    'reshape_as',
    ...
]
```
The downside of this is that when you're done adding operators this list is going to be 600 lines long